frameworks/elastic/build.gradle

```diff
@@ -9,21 +9,18 @@ repositories {
     mavenLocal()
     mavenCentral()
     maven {
-        url "http://downloads.mesosphere.com/maven/"
+        url "https://s3-us-west-1.amazonaws.com/px-dcos/maven/"
     }
     maven {
-        url "http://downloads.mesosphere.com/maven-snapshot/"
-    }
-    maven {
-        url 'http://repository.apache.org/content/repositories/snapshots/'
+        url "https://s3-us-west-1.amazonaws.com/px-dcos/maven-snapshot/"
     }
 }
 
 ext {
     junitVer = "4.12"
     systemRulesVer = "1.16.0"
     mockitoVer = "2.10.0"
-    dcosSDKVer = "0.56.2"
+    dcosSDKVer = "0.40.5-1.3.4"
```

Comment:
Can we use the latest here?
0.40.5-1.3.7

Reply:
Changed.